README.md

```diff
@@ -743,6 +680,16 @@ Future<void>.delayed(Duration.zero, () {
 });
 ```
 
+## Push Notifications
+
+By default, push-related components in the sample app won't work on Android, because of a dummy [google-services.json](example/android/app/src/google-services.json) file. In order to use push messaging features of Ably SDK, additional FCM/APNS configuration is required.
+
+See [PushNotifications.md](PushNotifications.md) for detailed information on getting PN working with the example app.
+
+## Feature support
```

Comment:
Let's delete this section, support matrix is outdated and link is broken

Reply:
Got it

Reply:
Seems this will be covered as a part of https://github.com/ably/ably-flutter/pull/515